cdap-docs/reference-manual/source/http-restful-api/artifact.rst

```diff
@@ -197,6 +197,191 @@ artifact (pretty-printed and reformatted to fit):
       "version": "|release|"
     }
 
+.. _http-restful-api-artifact-set-properties:
+
+Set Artifact Properties
+=======================
+To set properties for a specific version of an artifact, submit an HTTP PUT request::
+
+  PUT <base-url>/namespaces/<namespace>/artifacts/<artifact-name>/versions/<artifact-version>/properties
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<namespace>``
+     - Namespace ID
+   * - ``<artifact-name>``
+     - Name of the artifact.
```

Comment:
I notice that we are inconsistent in the punctuation of these descriptions. I think our style (be that it may be called a style) is to only have a trailing period if there is more than one sentence in the description.